docs/learn/web-development/es-modules.md

```diff
@@ -49,7 +82,7 @@ Both [@web/test-runner](../../docs/test-runner/overview.md) and [@web/dev-server
 
 ## CommonJS modules
 
-CommonJS is the module system of NodeJS and predates the standard es module system. You can recognize them by the use of `require` and `module.exports`:
+CommonJS is the original module system of NodeJS and predates the standard es module system. You can recognize them by the use of `require` and `module.exports`:
```

Comment:
```suggestion
CommonJS is the original module system of NodeJS and predates standard es modules. You can recognize them by the use of `require` and `module.exports`:
```
This reads weird to me, but might not actually need changing.

Reply:
Adjusted the wording slightly.